Classes/IndexQueue/AbstractIndexer.php

```diff
@@ -187,6 +190,9 @@ protected function resolveFieldValue(
                 $fieldValue = unserialize($fieldValue);
             }
         } else {
+            if (!array_key_exists($indexingConfiguration[$solrFieldName], $data)) {
```

Comment:
If think we should also check `$indexingConfiguration[$solrFieldName]` , a non-empty string is required

E.g.:

```
if (is_string($indexingConfiguration[$solrFieldName] ?? null)  && !array_key_exists($indexingConfiguration[$solrFieldName], $data))
```